client/shared/src/api/client/mainthread-api.ts

```diff
@@ -48,9 +48,20 @@ export const initMainThreadAPI = (
             .subscribe()
     )
 
-    // Commands
     const api: MainThreadAPI = {
         applySettingsEdit: edit => updateSettings(platformContext, edit),
+        requestGraphQL: <TResult extends object = any, TVariables extends object = any>(
+            request: string,
+            variables: TVariables
+        ) =>
+            from(
+                platformContext.requestGraphQL<TResult, TVariables>({
+                    request,
+                    variables,
+                    mightContainPrivateInfo: true,
+                })
+            ).toPromise(),
```

Comment:
Part of above suggestion + `from()` is not needed
```suggestion
        requestGraphQL: (request, variables) =>
                platformContext.requestGraphQL({
                    request,
                    variables,
                    mightContainPrivateInfo: true,
                }).toPromise(),
```